01-path-basics/101-start-here/templates/lab-ide-vpc.template

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Do we need this file any more?

Reply:
For now, yes as users have the option of deploying Cloud9 either in a VPC or outside of a VPC.

Either way, it will build a new VPC for EKS (Separate from Cloud9).